Portal/src/Datahub.Portal/Pages/Help/HelpPage.razor

```diff
@@ -121,7 +121,7 @@
                             </MudPaper>
                         }
                         <MudGrid Class="pt-3">
-                            <MudItem xs="6" align="@Align.Start" Class="pb-6">
+                            <MudItem xs="6" align="@Align.Start" class="pb-2">
```

Comment:
`Class` is an attribute on the `MudItem` blazor component and `class` will be attribute splattered below. Are you sure this is intended?

Either way it should be consistent with the `MudItem` on L134

Reply:
Not intended, good catch